### PR TITLE
replenish e2e test for kubelet instance config

### PR DESCRIPTION
**Pull request by HirazawaUi**
Add tests for https://github.com/kubernetes/kubernetes/pull/129278 and https://github.com/kubernetes/kubernetes/pull/129279

### CODE REVIEW

File: kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml [hunk 1]
```diff
@@ -218,6 +218,36 @@ tasks:
         exit 1;
         }
 
+      exit 0
+  timeout: 5m
+- name: check-and-add-kubeadm-flags
```
Comment (neolit123) [inline kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml:223]:
```suggestion
- name: check-and-add-flag
```

**Review by neolit123 — COMMENTED** [review]
just one nit about task name but LGTM

File: kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml (hunk 1 above)
Comment (HirazawaUi) [reply kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml]: fixed.

Comment (neolit123) [reply kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml]: i recommended `flag` not `flags`.
in follow up tasks you have `check-flag-after-second-upgrade`.

just try makaing the naming consistent.

Comment (HirazawaUi) [reply kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml]: Thank you for your carefulness, fixed again.

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm
/approve

**Pull request merged**